airflow/utils/dag_processing.py

```diff
@@ -388,12 +415,20 @@ def _run_processor_manager(dag_directory,
                                processor_factory,
                                processor_timeout,
                                signal_conn,
-                               async_mode):
+                               dag_ids,
+                               pickle_dags,
+                               async_mode,
+                               inherited_conf=None):
 
         # Make this process start as a new process group - that makes it easy
         # to kill all sub-process of this at the OS-level, rather than having
         # to iterate the child processes
         os.setpgid(0, 0)
+        if inherited_conf is not None:  # pylint: disable=too-many-nested-blocks
+            for section in inherited_conf:
+                for key, value in inherited_conf[section].items():
+                    if value not in conf:
+                        conf.set(section, key, value.replace("%", "%%"))
```

Comment:
This looks suspect. What's going on here?

Reply:
I also don't see anywhere that we actually pass a value here?

Reply:
`spawn` method doesn't inherit environment except for a little bit of things so `conf` is not also inherited by child process. I noticed `test_reload_module` in `test_dag_processing.py` fails because `DAG_PROCESSOR_MANAGER_LOG_LOCATION` configured is not inherited.

Reply:
Sorry, not `DAG_PROCESSOR_MANAGER_LOG_LOCATION` but `logging_config_class`.

Reply:
@ashb If only `test_reload_module` needs to dynamically change the configuration and propagate it to child processes, I'll remove `inherited_conf` and its related code.
What do you think?